boms/cloud-oss-bom/pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>4.2.1-SNAPSHOT</version>
+  <version>4.3.0-SNAPSHOT</version>
```

Comment:
@elharo I see the gRPC version change has been removed from this PR. You may want to exclude this version change too.

Reply:
I'm assuming the next version change will pick up a minor, not patch, release but if not we can always change this again.